scripts/build_script.sh

```diff
@@ -15,6 +15,7 @@ help()
     echo "h     Print this Help."
     echo "p     Generate a Debian package using CPack."
```

Comment:
It seems like there is not a CMake flag "CREATE_DEB_PACKAGE" in addition to this "p" argument for generating a debian package. Can one of them be removed?

Reply:
I can remove the parameter in the carma-time-lib. We can probably handle that in a separate PR.